xc/stubdom.ml

```diff
@@ -77,9 +77,10 @@ let build (task: Xenops_task.task_handle) ~xc ~xs ~dm ~store_domid ~console_domi
   let path = Printf.sprintf "/vm/%s/image/dmargs" guest_uuid_str in
   (* Remove any 'pty' references from the arguments: XXX why? *)
   let info' = { info with Device.Dm.serial = None; monitor = None } in
-  let args = Device.Dm.cmdline_of_info ~xs ~dm info' false domid in
-  xs.Xs.write path (String.concat " " args);
-  debug "jjd27: written qemu-dm args into xenstore at %s: [%s]" path (String.concat " " args);
+  let args = Device.Dm.qemu_args ~xs ~dm info' false domid in
+  let argv = args.Device.Dm.argv in
+  xs.Xs.write path (String.concat " " argv);
+  debug "jjd27: written qemu-dm args into xenstore at %s: [%s]" path (String.concat " " argv);
```

Comment:
We shouldn't have user specific indicators (jjd27) in the debug messages.

Reply:
I agree. This code was already there but I can take it out.